main.go

```diff
@@ -114,6 +115,13 @@ func processLine(line string, config *Config) {
 		}
 	}
 
+	// parse our json
+	if config.jsonSet {
+		if jsonRegex.MatchString(log.values[config.MessageKey]) {
```

Comment:
idk if performance matters here but im wondering how slow regex is vs https://github.com/valyala/fastjson

Reply:
even faster might be peek at the first and last runes of the string

Reply:
added rune check